manning/manning-proposal-2020-06.md

```diff
@@ -0,0 +1,204 @@
+6.  Adopt a license, a code of conduct, and efficient decision-making processes to ensure the project runs smoothly.
+7.  Publish results with DOIs, ORCIDs, and other metadata in a fully-reproducible way.
+
+**4. The minimally-qualified reader (MQR)**
+
+*Amira Khan*
+completed a master's in library science five years ago
+and has since worked for a small aid organization.
+She did some statistics during her degree,
+and has learned some R and Python by doing data science courses online,
+but has no formal training in programming.
+Amira would like to tidy up the scripts, data sets, and reports she has created
+in order to share them with her colleagues.
+These lessons will show her how to do this and what "done" looks like.
+
+*Jun Hsu*
+completed a 4-month data science bootcamp last year after doing a PhD in Geology
+and now works for a company that does forensic audits.
+He uses a variety of machine learning and visualization packages,
+and would now like to turn some of his own work into an open source project.
+This book will show him how such a project should be organized
+and how to encourage people to contribute to it.
+
+*Sami Virtanen*
+became a competent programmer during a bachelor's degree in applied math
+and was then hired by the university's research computing center.
+The kinds of applications they are being asked to support
+have shifted from fluid dynamics to data analysis;
+this guide will teach them how to build and run data pipelines
+so that they can pass those skills on to their users.
+
+**5. Q&A**
+
+*What are the three or four most commonly-asked questions about this technology?*
+
+We wrote this book in part because the people who need it don't even know what questions to ask.
+"How do I write a general rule in a Makefile?" presupposes that you know that Make exists.
+Similarly,
+people don't ask, "How should a small group run a meeting?"
+because it doesn't occur to them that running meetings is an improvable skill.
+
+*What are others interested in this topic asking in forums?*
+
+All of our chapters are driven by questions we have answered (some of them many times)
+in workshops or in discussions with colleagues.
+Our answers for technical subjects frequently draw on Stack Overflow,
+particularly for topics like continuous integration and packaging;
+our answers for community management and project leadership draw on extensive experience in open source.
+
+**6. Tell us about the competition and the ecosystem.**
+
+*What other books are available on this topic?*
+
+*The Turing Way* (https://the-turing-way.netlify.app/), produced by the Turing Institute in the UK,
+is the only up-to-date resource we know of with similar breadth.
+Other books cover only one topic in more depth than our audience needs
+(e.g., Ray & Ray's *Unix and Linux: Visual QuickStart Guide*)
+or are aimed at commercial software developers
+(any number of books on testing, coding style, and dev ops).
+
+*How does the proposed book compare to them?*
```

Comment:
Points of difference:
1. Our focus on programming best practices (e.g. principles like writing modular, reusable, testable code). Others teach basic Python and R syntax and how to complete isolated tasks with particular libraries, but in this day and age you can pick up that information from a Google search. Our best practice content is the bigger picture you can't glean from online code snippets or Stack Overflow answers.
2. We go through the process of building a data science workflow/package from scratch. Most other texts simply present information (like a reference book) - we tackle a real data science problem from beginning to end, weaving in relevant information as we go.
3. Our book is represents a ready to go university semester course. There is growing demand for Software Carpentry the semester course.

Reply:
:+1: